Cargo.toml

```diff
@@ -219,48 +219,48 @@ members = [
 #
 # This list is ordered alphabetically.
 [profile.dev.package]
-blake2 = { opt-level = 3 }
-blake2-rfc = { opt-level = 3 }
-blake2b_simd = { opt-level = 3 }
-chacha20poly1305 = { opt-level = 3 }
+blake2            = { opt-level = 3 }
```

Comment:
personally I don't like aligning, it makes every line long when only one crate has a big name.